packages/error-reporting/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@google-cloud/error-reporting",
+  "description": "Stackdriver Error Reporting Client Library for Node.js",
+  "main": "index.js",
+  "repository": "GoogleCloudPlatform/google-cloud-node",
+  "scripts": {
+    "test": "nyc --exclude=\"fuzzer.js\" $(npm bin)/mocha ./test/unit/*.js",
+    "system-test": "nyc --exclude=\"error-message.js\" $(npm bin)/mocha ./test/system-test/*.js",
+    "lint": "jshint src/ index.js",
+    "docs": "jsdoc -d docs index.js src/",
```

Comment:
We have our own docs script which knows how to chew through source files and merge it all up into our docs site. If we want your docs to exist on this gh-pages branch, we'll have to work with @callmehiphop to sync these up.